Gemfile.lock

```diff
@@ -21,7 +21,7 @@ PATH
       faraday-retry
       faye-websocket
       filesize
-      hrr_rb_ssh-ed25519
```

Comment:
This was swapped as part of https://github.com/rapid7/metasploit-framework/pull/15493/commits/ab31500d7f0c413a93eacd713b39b3e694114b04

It looks like the ilbrary should just be an extension to `hrr_rb_ssh` - I might need to dig a bit deeper on why the addition of ed25519 causes it to not work anymore.